test/_e2e/cleaner.go

```diff
@@ -0,0 +1,29 @@
+package e2e
+
+import "strings"
+
+// CleanOutput Some commands, such as deploy command, spans spinner chars and cursor shifts at output which are captured and merged
+// regular output messages. This functions is meant to remove these chars in order to facilitate tests assertions and data extraction from output
+func CleanOutput(stdOutput string) string {
```

Comment:
I think this might be:
```go
func CleanOutput(stdOutput string) string {
	var re = regexp.MustCompile(`([🕛🕐🕑🕒🕓🕔🕕🕖🕗🕘🕙🕚] |\33\[1A|\33\[1B|\33\[K)`)
	return re.ReplaceAllLiteralString(stdOutput, "")
}
```

Reply:
but doesn't really matter